tasks/cluster_agent.py

```diff
@@ -5,6 +5,7 @@
 import os
 import glob
 import shutil
+from distutils.dir_util import copy_tree
```

Comment:
where is `copy_tree` used?

Reply:
it's used in the next PRs in the `refresh_asset` method. I can remove for now.